docs/precommit.md

```diff
@@ -66,9 +66,10 @@ Read more at [mirror of prettier package for pre-commit](https://github.com/pre-
 **Use Case:** A dotnet solution to use Prettier along with other code quality tools (e.g. dotnet-format, ESLint, Stylelint, etc.). It supports multiple file states (staged - last-commit, git-files etc.)
 
 ```bash
+dotnet new tool-manifest
 dotnet tool install husky
 dotnet husky install
-dotnet husky add .husky/pre-commit
+dotnet husky add pre-commit
```

Comment:
Do we need  `dotnet husky add pre-commit -c "dotnet husky run"`?  https://alirezanet.github.io/Husky.Net/guide/task-runner.html#task-runner-json

Reply:
I did not need the extra parameters.
Just tested it by starting with an empty folder and pre commit runs fine.
Apparently the "dotnet husky run" is implied. You could add something else, but then the Husky.Net built-in task runner will not run.

![image](https://user-images.githubusercontent.com/352626/203346499-485a013f-20ff-47e1-a530-8ad6ee59b57a.png)


![image](https://user-images.githubusercontent.com/352626/203347735-f038b19e-4ab1-44c1-8a89-7e09aa8cffd3.png)